frontend/src/components/common/RenderCheckbox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Checkbox } from 'antd';
+
+/**
+ * @constant RenderCheckbox - Ant Design `Checkbox` component for redux-form.
+ */
+
+const propTypes = {
+  id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  label: PropTypes.string,
+  meta: PropTypes.object,
+};
+
+const RenderCheckbox = ({
+  id,
+  input,
+  label,
+  meta: { touched, error},
+}) => (
+  <Form.Item 
+  validateStatus={(touched ? (error && 'error') : '')}
+>
+  <Checkbox 
+    id={id} 
+    checked={input.value || false}
```

Comment:
Is the `|| false` necessary here? My impression would be that `input.value` gets coerced?

But if it doesn't, how about `checked={Boolean(input.value)}` to be a bit more explicit about the issue?

Reply:
false is not needed.